src/service/CPUEnergyService6D.tpp

```diff
@@ -170,6 +170,18 @@ auto CPUEnergyService6D<REAL>::createItemProcessor() -> itemProcessor_t {
 					lig->numAtoms()
 			); // OK
 
+			reduceModeForce(
```

Comment:
this function shouldn't be in this file (but in CPUEnergyService6DModes.tpp)
)

Reply:
true